.github/workflows/build.yml

```diff
@@ -9,17 +9,23 @@ jobs:
       BASEDIR: ${{github.workspace}}/arrow-libs
       JAVA_OPTS: -Xms512m -Xmx1024m
 
-    runs-on: macos-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest]
+        java: [8, 11, 13, 15]
+    name: Build libraries with JDK ${{ matrix.java }} on ${{ matrix.os }}
```

Comment:
Wondering if we want to build on all these versions. I was thinking about splitting github actions to build/test jvm and javascript in parallel. Since the build went up considerable now we added MPP.

Before 4-5 minutes for a build, now ~20 minuntes.
But for every MPP library we generate 4 JS binaries  + JVM, and soon native which has a lot of targets as well.

How does this impact CI time? How many Github Actions can we run in parallel? 🤔

Reply:
I think they are run in Parallel as far as I can tell, so we should be safe :crossed\_fingers:

Reply:
Just had to check and it seems that way :partying\_face: 

![](https://user-images.githubusercontent.com/260697/125120753-d2e31a80-e0ea-11eb-8298-1970f66ba49a.gif)